src/Quizzer.API.DAL/QuizzerDbContext.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Quizzer.API.DAL.Entities;
+
+namespace Quizzer.API.DAL {
+    public class QuizzerDbContext : DbContext {
+        public DbSet<QuizEntity> Quiezzes { get; set; } = null!;
+        public DbSet<CompletedQuizEntity> CompletedQuiezzes { get; set; } = null!;
+        public DbSet<QuestionEntity> Questions { get; set; } = null!;
+        public DbSet<AnswerEntity> Answers { get; set; } = null!;
+        public DbSet<SelectedAnswerEntity> SelectedAnswers { get; set; } = null!;
+        public DbSet<UserEntity> Users { get; set; } = null!;
+        public DbSet<UserAnswerEntity> UserAnswers { get; set; } = null!;
```

Comment:
This should not be null, instead initialize it with Set method

Reply:
done